docs/product_search/parameters.md

```diff
@@ -0,0 +1,240 @@
+Will return experiences which depart within the [departure date]({% link docs/reference/objects/product/index.md %}#productdepart_on) range.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search departure_date: {greater_or_equal_than: 'now', less_than: '2023-12-28' } %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as query paramameters
+```
+https://mysite.com/search?search[departure_date][greater_or_equal_than]=2022-11-19&search[departure_date][less_than]=2023-12-28
+```
+
+### departure_month
+Accepts a month as either a 3 letter abreviation, or the month number i.e. `Apr` or `4`.<br>
+Will return experiences which [depart]({% link docs/reference/objects/product/index.md %}#productdepart_on) within the month passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search departure_month: 'Apr' %}
+{% endproduct_search %}
+
+{% product_search departure_month: 4 %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as query paramameters
+```
+https://mysite.com/search?search[departure_month]=Apr
+https://mysite.com/search?search[departure_month]=4
+```
+
+### duration
+Accepts an object which specifies how to handle the search, through `equal_to` `greater_than` or `less_than` each taking a number of days.<br>
+Will return experiences which have a [duration]({% link docs/reference/objects/product/index.md %}#productduration) within the defined range.<br>
+Note: Experience [durations]({% link docs/reference/objects/product/index.md %}#duration) can be returned as a number of hours, 1 day or a number of nights, whereas product_search `duration` always takes the duration as a number of days.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search duration: {greater_than: 3, less_than: 8 } %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as query paramameters
+```
+https://mysite.com/search?search[duration][greater_than]=3&search[duration][less_than]=8
+```
+
+### exclude_sold_out_products
+Accepts: `true` or `false`.<br>
+Passing `true` will exclude [sold out]({% link docs/reference/objects/product/index.md %}#productsold_out) products from the products returned.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search exclude_sold_out_products: true %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[exclude_sold_out_products]=true
+```
+
+### include_organisation_products
+Accepts: `true` or `false`.<br>
+Passing `true` will include products from other companies which are linked in the same organisation.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search include_organisation_products: true %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[include_organisation_products]=true
```

Comment:
I'm getting a liquid error trying to use this one. 
But I do get results with e.g. https://www.bayou-retreats.com/search-filter?search[include_organisation_products]

Reply:
It looks like if the company isn't part of an organisation this returns a liquid error, have created a ticket for this one.
Example:  I've added this company to an organisation and this now works https://sites-xnmsj.myeasol.com/search-filter?search[include_organisation_products]=true